pkg/nsxt/connector_manager.go

```diff
@@ -194,6 +194,9 @@ func (cm *ConnectorManager) GetConnector() client.Connector {
 
 // AddSecretListener adds secret informer add, update, delete callbacks
 func (cm *ConnectorManager) AddSecretListener(secretInformer v1.SecretInformer) error {
+	if cm.config == nil {
```

Comment:
@andrewsykim received nil pointer errors if the config is not set, thinking we should add this here, or should I open another PR for this fix?